src/web/src/components/SignUp/Schema/FormModel.tsx

```diff
@@ -0,0 +1,55 @@
+export default {
+  displayName: {
+    name: 'displayName',
+    label: 'Display Name',
+    requiredErrorMsg: 'Display Name is required',
```

Comment:
For consistency with the errors below, `Display name` (lower case `name`).

Reply:
Done